MKOB/kobmain.py

```diff
@@ -175,23 +208,16 @@ def readerCallback(char, spacing):
         ka.codereader_append("\n")
 
 def reset_wire_state():
-    """log the current latching states and reinitialize"""
-    global kob_latched, keyboard_latched, internet_latched
-    print("Reset\n kob_latched: {}\n keyboard_latched: {}\n internet_latched: {}"
-            .format(kob_latched, keyboard_latched, internet_latched))
-    kob_latched = True
-    keyboard_latched = True
-    internet_latched = True
+    """log the current internet state and regain control of the wire"""
+    global internet_active
+    print(
```

Comment:
Would you want to use `log.info(...)` here. We kind of got away from using the Log, but we should probably go back to it. We can add functionality to 'Log' over time to be able to log to a file, to the console, to both, etc. It currently adds a timestamp, which can come in handy.

Reply:
I'm on the verge of deleting that print statement and not logging the information.

Reply:
If you want to that's fine. 

At some point I expect we will incorporate a logging package. They generally provide a lot of control about what is and isn't output to the log (without changing any code) and most even allow you to change the verbose-ness dynamically while the app is running. Since we don't have that in place yet, if you don't want to clutter the output you can remove it. It didn't seem to occur too often, so it didn't bother me, though I can see it being better as `log.debug(...)`.